docs/_rpc/ns-debug.md

```diff
@@ -824,10 +825,12 @@ If the step function throws an exception or executes an illegal operation at any
 - `to` - Address, target of the transaction
 - `input` - Buffer, input transaction data
 - `gas` - Number, gas budget of the transaction
+- `gasUsed` - Number, amount of gas used in executing the transaction (excludes txdata costs)
+- `gasPrice` - Gas price configured in the transaction being executed
+- `intrinsicGas` - Intrinsic gas for the transaction being executed
 - `value` - big.Int, amount to be transferred in wei
 - `block` - Number, block number
 - `output` - Buffer, value returned from EVM
-- `gasUsed` - Number, amount of gas used in executing the transaction (excludes txdata costs)
 - `time` - String, execution runtime
```

Comment:
Can you please also add `blockHash`, `txIndex` and `txHash` which are mined when tracing a mined tx (i.e. `traceCall` leaves them empty).

Reply:
Added, can you check again?